src/test/java/com/google/api/codegen/testdata/discoveries/csharp/csharp_appengine.v1beta5.json.baseline

```diff
@@ -17,6 +17,7 @@
 using Google.Apis.Appengine.v1beta5;
 using Google.Apis.Auth.OAuth2;
 using Google.Apis.Services;
+using Newtonsoft.Json;
```

Comment:
@chrisdunelm 

In the header above, in step 3 we tell users how to add the `Google.Apis` dependency (honestly, looking at it now, I think we could be a bit more clear...).
Do I need to mention `Newtonsoft.Json` as well? I noticed when I ran this sample that I didn't need to add it to `project.json`.